docs/src/main/paradox/configuration.md

```diff
@@ -145,4 +145,4 @@ pekko-persistence-jdbc {
 ## Explicitly shutting down the database connections
 
 The plugin automatically shuts down the HikariCP connection pool when the ActorSystem is terminated.
-This is done using @apidoc[ActorSystem.registerOnTermination](ActorSystem).
+This is done using @apidoc[org.apache.pekko.actor.ActorSystem.registerOnTermination](org.apache.pekko.actor.ActorSystem).
```

Comment:
This shouldn't be a problem, if it was, we could optimize it as:

```suggestion
This is done using @apidoc[ActorSystem.registerOnTermination](org.apache.pekko.actor.ActorSystem).
```

Reply:
was causing an issue when I added a dependency on pekko-persistence-typed but that dependency is gone now so this is not needed